x/incentives/client/cli/tx.go

```diff
@@ -128,3 +132,136 @@ func NewAddToGaugeCmd() *cobra.Command {
 		Short: "add coins to gauge to distribute more rewards to users",
 	})
 }
+
+// NewCmdHandleCreateGaugeGroupsProposal implements a command handler for the gauge group proposal transaction.
+func NewCmdHandleCreateGaugeGroupsProposal() *cobra.Command {
```

Comment:
Ditto for renaming GaugeGroups to simply Groups as well as everywhere below

Reply:
https://github.com/osmosis-labs/osmosis/pull/6547/commits/cb2b2a0756528d492c159ed468ef42d763c51cb3